tests/test_investpy.py

```diff
@@ -159,6 +159,11 @@ def test_investpy_stocks():
             'as_json': False,
             'n_results': 50
         }
+        {
+            'country': 'indonesia',
+            'as_json': False,
+            'n_results': 362
```

Comment:
Why 362 results? Is that the only way to trigger the billion-option parameter?

Reply:
yeah, it's in 362rd element in array

Reply:
but i think for the next development you should make a stub or mock the call to investing.com into some json result

Reply:
OK cool! Then I'll merge this now, feel free to create any new PR improving the tests or whatever you feel to 😄